octue/cloud/pub_sub/service.py

```diff
@@ -128,13 +128,13 @@ def serve(self, timeout=None, delete_topic_and_subscription_on_exit=False, allow
         :return (google.cloud.pubsub_v1.subscriber.futures.StreamingPullFuture, google.cloud.pubsub_v1.SubscriberClient):
         """
         logger.info("Starting %r.", self)
-
         topic = Topic(name=self._pub_sub_id, project_name=self.backend.project_name)
 
         subscription = Subscription(
             name=self._pub_sub_id,
             topic=topic,
             project_name=self.backend.project_name,
+            filter='attributes.is_question = "1"',
```

Comment:
We could use `attributes.sender="parent"` instead if we wanted to be ready to generalise to parents being able to send more than one message.

Reply:
I've gone for `sender_type`